perfzero/lib/perfzero/utils.py

```diff
@@ -242,7 +243,7 @@ def run_command(cmd, shell=True):
   Returns:
     Tuple of the command return value and the standard out in as a string.
   """
-  logging.debug('Executing command: {}'.format(cmd))
+  logging.debug('Executing command: %s', cmd)
```

Comment:
nits: Not sure if there is specific benefit of using `%s` instead of `{}`. Would it be a bit better to use the same string formatting style across the project?

Reply:
logging.debug is "supposed" (I mean this weakly) to be used this way or at least that is what pylint says.  I also have changed most of our tensorflow logging to do it this way.  If this was a print I would 100% agree.